presto-orc/src/main/java/com/facebook/presto/orc/reader/MapStreamReader.java

```diff
@@ -128,8 +126,8 @@ public Block readBlock(Type type)
         Type valueType = mapType.getValueType();
 
         int entryCount = 0;
-        for (int length : lengthVector) {
-            entryCount += length;
+        for (int i = 0; i < offsetVector.length - 1; i++) {
+            entryCount += offsetVector[i];
```

Comment:
nit: Add a comment here `offsetVector` still stores length (instead of offset).

This is because if someone happened to read this part of code, he/she will be very confused without looking back to the comments at line 104.

Reply:
sure